ui/package.json

```diff
@@ -27,6 +27,11 @@
     "@splunk/splunk-utils": "^3.0.0",
     "@splunk/themes": "^0.17.0",
     "@splunk/ui-utils": "^1.6.0",
+    "@splunk/dashboard-context": "^26.6.0",
+    "@splunk/dashboard-action-buttons": "^26.6.0",
+    "@splunk/visualization-context": "^25.13.1",
+    "@splunk/dashboard-core": "^26.6.0",
+    "@splunk/dashboard-presets":"^26.6.0",
```

Comment:
@soleksy-splunk 
does it work with the latest?
```
    "@splunk/dashboard-context": "^26.6.3",
    "@splunk/dashboard-action-buttons": "^27.0.0",
    "@splunk/visualization-context": "^26.0.0",
    "@splunk/dashboard-core": "^27.0.0",
    "@splunk/dashboard-presets": "^27.0.0",
```

We requested to clear out fossa alerts, I don't want to repeat this after the next npm update...

Reply:
warning " > @splunk/dashboard-action-buttons@27.0.0" has unmet peer dependency "@splunk/dashboard-state@^27.0.0".
do you want to add this as well?

Reply:
@vtsvetkov-splunk , yea definitely make sense to bump it to newest version, let me check if all works as expected with 27, as it should be relatively new one (when we added 26.6 was newest)

Reply:
What is about this one?
```
warning " > @splunk/dashboard-action-buttons@27.0.0" has unmet peer dependency "@splunk/dashboard-state@^27.0.0".
warning "@splunk/dashboard-action-buttons > @splunk/dashboard-ui@27.0.0" has unmet peer dependency "@splunk/dashboard-state@^27.0.0".
warning " > @splunk/dashboard-core@27.0.0" has unmet peer dependency "@splunk/dashboard-state@^27.0.0".
```

Reply:
@vtsvetkov-splunk  i updated to the newest, and seems to work the same, not sure where did you found that warning, but if thats connected with fossa worth adding it with one request.

Reply:
just to remove that warning i added it as separate dependency